src/main/java/com/rakovets/course/java/core/practice/operators/Task01.java

```diff
@@ -29,8 +29,7 @@ public static void main(String[] args) {
      * @param carryingCapacity грузоподъемность грузовиков
      */
     static int getNumberFullTrucks(int cargo, int carryingCapacity) {
-        //TODO
-        // Код, решающий задачу пишем ниже, при этом используя параметры метода
-        return 0;
+       int totalTrucks = cargo / carryingCapacity;
```

Comment:
"Violation: There aren't correct indentations for inner code
Solution: Add correct indentations for inner code"